### ISSUE TITLE
publish standard macos package release media

### ISSUE THREAD

**Issue by mcandre** [scrod/nv#393]
For non-developer consumption.

Example: .PKG installer, .DMG disk image. Or a zipped .APP. Or at least a zipped executable.

**Comment by chengguangnan** [comment]
Yes, I'm hoping for the same thing. The download link on https://notational.net/ is broken.

**Comment by chengguangnan** [comment]
By google "NotationalVelocity.zip", I found an old back up from https://www.macintoshrepository.org/

**Comment by jptechco** [comment]
If you want a universal package that works on both Apple Silicon and Intel platforms - check out https://kineticnotes.org
